sympy/matrices/matrices.py

```diff
@@ -1270,24 +1270,60 @@ def eigenvals(self, error_when_incomplete=True, **flags):
     def eigenvects(self, error_when_incomplete=True, iszerofunc=_iszero, **flags):
         """Return list of triples (eigenval, multiplicity, basis).
 
-        The flag ``simplify`` has two effects:
+        Parameters
+        ==========
+
+        error_when_incomplete : bool, optional
+            Raise an error when not all eigenvalues are computed. This is
+            caused by ``roots`` not returning a full list of eigenvalues.
+
+        iszerofunc : function, optional
+            Specifies a zero testing function to be used in ``rref``.
+
+            Default value is ``_iszero``, which uses naive and fastest approach
+            by using sympy's default assumption handler.
+
+            It can also accept any user-specified zero testing function, if it
+            is formatted as a function which accepts a single symbolic argument
+            and returns ``True`` if it is tested as zero and ``False`` if it
+            is tested as non-zero, and also ``None`` if it is undecidable.
+
+        simplify : bool or function, optional
             1) if bool(simplify) is True, as_content_primitive()
             will be used to tidy up normalization artifacts;
+
             2) if nullspace needs simplification to compute the
             basis, the simplify flag will be passed on to the
             nullspace routine which will interpret it there.
 
-        Parameters
-        ==========
+        chop : bool or positive number, optional
+            If the matrix contains any Floats, they will be changed to Rationals
+            for computation purposes, but the answers will be returned after
+            being evaluated with evalf. If it is desired to removed small
+            imaginary portions during the evalf step, pass a value for the
+            ``chop`` flag.
 
-        error_when_incomplete : bool
-            Raise an error when not all eigenvalues are computed. This is
-            caused by ``roots`` not returning a full list of eigenvalues.
+        Returns
+        =======
+        ret : [(eigenval, multiplicity, [eigenvector, ...]), ...]
+            A ragged list containing tuples of data obtained by ``eigenvals``
+            and ``nullspace``.
+
+            ``eigenvector`` is a vector in the form of the ``Matrix``.
+            For example, ``Matrix([[a_1], [a_2], [a_3]])`` if it is a vector of
```

Comment:
I have used double brackets because the return format is actually like.
Though I think either can be okay.
```
>>> from sympy import *
>>> Matrix([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0], [7.0, 8.0, 9.0]]).eigenvects()
[(0, 1, [Matrix([
   [ 1.0],
   [-2.0],
   [ 1.0]])]), (16.1168439698070, 1, [Matrix([
   [0.28334945180064],
   [0.64167472590032],
   [             1.0]])]), (-1.11684396980704, 1, [Matrix([
   [-1.28334945180064],
   [-0.14167472590032],
   [              1.0]])])]
```

Reply:
The extra square braces are not necessary when representing vectors.
(`Matrix([[a_1, a_2, a_3]])` is also essentially the same but the former is minimal and readable)